gui/src/ts/api/queries/benchmark/BenchmarkAnswerPageQuery.graphql

```diff
@@ -1,34 +1,40 @@
-query BenchmarkAnswerPageQuery($benchmarkId: String!, $questionId: String!, $datasetId: String!, $submissionId: String!) {
-    benchmarkById(id: $benchmarkId) {
-        name
-        datasetById(id: $datasetId) {
-            name
-            questionById(id: $questionId) {
-                choices {
-                    label
-                    text
-                }
-                text
-            }
-            submissionById(id: $submissionId) {
-                answerByQuestionId(id: $questionId) {
-                    choiceLabel
-                    explanation {
-                        choiceAnalyses {
-                            choiceLabel
-                            questionAnswerPaths {
-                                endNodeId
-                                startNodeId
-                                paths {
-                                    path
-                                    score
-                                }
-                            }
-                        }
-                    }
+query BenchmarkAnswerPageQuery(
+  $benchmarkId: String!
+  $questionId: String!
+  $datasetId: String!
+  $submissionId: String!
+) {
+  benchmarkById(id: $benchmarkId) {
+    name
+    datasetById(id: $datasetId) {
+      name
+      questionById(id: $questionId) {
+        choices {
+          label
+          text
+        }
+        text
+        correctChoiceLabel
+      }
+      submissionById(id: $submissionId) {
+        answerByQuestionId(id: $questionId) {
+          choiceLabel
+          explanation {
+            choiceAnalyses {
+              choiceLabel
+              questionAnswerPaths {
+                endNodeId
+                startNodeId
+                paths {
+                  path
```

Comment:
By the way, path has a synthesized "edges" field which will give you back KgEdges (which have redundancy because object becomes the next subject) instead of the compact path. But you can also use that to get node labels.

Reply:
Oh okay I'll take a look